R/SummaryPanel4a.R

```diff
@@ -0,0 +1,67 @@
+#' Module UI for the covariate summary plot tab.
+#' 
+#' @param id ID of the module
+#' @return Tab 4a Summary
+metaregression_summary_panel_ui <- function(id) {
+  ns <- NS(id)
+  fluidPage(
+    h4('Summary Characteristic Plot'),
+    uiOutput(ns('toggle_covariate_baseline')),
+    plotOutput(outputId = ns('covariate_plot')),
+    textOutput(outputId = ns('covariate_info')),
+    radioButtons(inputId = ns('format_covariate_plot'), 
+                 label = 'Document format', 
+                 choices = c('PDF' = 'pdf', 'PNG' = 'png'), 
+                 inline = TRUE),
+    downloadButton(outputId = ns('download_covariate_summary'))
+  )
+}
+
+#' Module server for the covariate summary plot tab.
+#'
+#' @param id ID of the module
+#' @param all_data Study data including covariate columns, in wide or long format
+#' @param metaoutcome Reactive containing meta analysis outcome: 'Continuous' or 'Binary'
+#' @param treatment_df Reactive containing data frame containing treatment IDs (Number) and names (Label)
+#' @return Covariate summary plot tab
```

Comment:
I don't think that there is a return from this function, so this can be removed from the roxygen.